### PR TITLE
add 1894 support to qmk

### PR DESCRIPTION
**Pull request by austexcn**
Adding support for the 1894 pcb. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/atxkb/1894/config.h [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x5458 // TX
+#define PRODUCT_ID      0x0001
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    atxkb
+#define PRODUCT         atxkb 1894
```
Comment (fauxpark) [inline keyboards/atxkb/1894/config.h:27]:
```suggestion
#define PRODUCT         1894
```

File: keyboards/atxkb/1894/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# atxkb 1894
+
+![atxkb 1894](https://i.imgur.com/epXYdPA.jpg)
+
+1894 is a keyboard PCB supporting 60% layouts. It is a revised version of the [Voyager60](https://github.com/ai03-2725/Voyager60) pcb by [ai03](https://github.com/ai03-2725) that supports a JST connector instead of usb-c and has perimeter cutouts for gasket support. The Voyager60 and therefore the 1894 is compatible with ai03's [polaris](https://github.com/austexcn/qmk_firmware/tree/master/keyboards/ai03/polaris) QMK implementation and as such this is just a copy with edits to update the name.
+
+Keyboard Maintainer: [austexcn](https://github.com/austexcn)  
+Hardware Supported: atxkb Moontower  
+Hardware Availability: [atxkb](https://atxkb.com)
```
Comment (fauxpark) [inline keyboards/atxkb/1894/README.md:9]:
```suggestion
* Keyboard Maintainer: [austexcn](https://github.com/austexcn)
* Hardware Supported: atxkb Moontower
* Hardware Availability: [atxkb](https://atxkb.com)
```

File: keyboards/atxkb/1894/rules.mk [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```
Comment (fauxpark) [inline keyboards/atxkb/1894/rules.mk:32]:
```suggestion
```

File: keyboards/atxkb/1894/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# atxkb 1894
```
Comment (fauxpark) [inline keyboards/atxkb/1894/README.md:1]: Please rename this and keyboards/atxkb/README.md to lowercase.

**Comment by austexcn** [comment]
@fauxpark thanks for the feedback! I went ahead and updated based on your changes on my end rather than committing the suggestions but they should all be taken care of now 👍

File: keyboards/atxkb/1894/readme.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# atxkb 1894
+
+![atxkb 1894](https://i.imgur.com/epXYdPA.jpg)
+
+1894 is a keyboard PCB supporting 60% layouts. It is a revised version of the [Voyager60](https://github.com/ai03-2725/Voyager60) pcb by [ai03](https://github.com/ai03-2725) that supports a JST connector instead of usb-c and has perimeter cutouts for gasket support. The Voyager60 and therefore the 1894 is compatible with ai03's [polaris](https://github.com/austexcn/qmk_firmware/tree/master/keyboards/ai03/polaris) QMK implementation and as such this is just a copy with edits to update the name.
+
+* Keyboard Maintainer: [austexcn](https://github.com/austexcn)  
+* Hardware Supported: atxkb Moontower
```
Comment (fauxpark) [inline keyboards/atxkb/1894/readme.md:8]:
```suggestion
* Keyboard Maintainer: [austexcn](https://github.com/austexcn)
* Hardware Supported: atxkb Moontower
```

**Comment by fauxpark** [comment]
Just one last thing, sorry.

**Comment by austexcn** [comment]
> Just one last thing, sorry.

no worries, updated!

File: keyboards/atxkb/1894/readme.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# atxkb 1894
+
+![atxkb 1894](https://i.imgur.com/epXYdPA.jpg)
```
Comment (noroadsleft) [inline keyboards/atxkb/1894/readme.md:3]: Swaps the image for a version that isn't so large.

```suggestion
![atxkb 1894](https://i.imgur.com/epXYdPAl.jpg)
```

**Comment by noroadsleft** [comment]
LOL, I was a bit too slow.

**Pull request merged**